### ISSUE TITLE
feedback link is wrong

### ISSUE THREAD

**Issue by tamptus** [Libre-TrainSim/Libre-TrainSim#551]
https://www.libre-trainsim.de/feedback
Server Error
404
Page Not Found

**Comment by Jomosoto** [comment]
Hi thanks for reporting. The page has moved to https://www.libretrainsim.org/feedback/, the button is updated in the next release. @Libre-TrainSim/dev what is the state of the old domain? Is it easy to create a redirect?

**Comment by HaSa1002** [comment]
Only Jean has access to the domains

Gesendet von meinem Windows 10 Mobile
________________________________

**Comment by HaSa1002** [comment]
@Jean28518 maybe you can do something about that?

**Comment by vikiawv** [comment]
It is already fixed

**Comment by Jomosoto** [comment]
Yes, but only in the developement version, in the last stable release it's still the old link.

**Comment by HaSa1002** [comment]
Fixed in v0.8.1
